plugins/languages/qmk.nix

```diff
@@ -15,7 +15,8 @@ helpers.neovim-plugin.mkNeovimPlugin config {
 
   settingsOptions = {
     name = mkOption {
-      type = types.str;
+      type = with types; nullOr str;
+      default = null;
```

Comment:
This seems unrelated?

Reply:
Because of the current implementation, documentation can't be built if any `plugins.*.settings.option` is missing a default value. This was the only error I found.
Now that I think of it, this limitation is invasive to all other plugin definitions. I should try to remove it.